optbinning/scorecard/scorecard.py

```diff
@@ -539,6 +547,11 @@ def _fit(self, df, metric_special, metric_missing, show_digits,
             binning_table["Variable"] = variable
             binning_table["Coefficient"] = c
             binning_table["Points"] = binning_table[bt_metric] * c
+            if metric_special != 'empirical':
+                binning_table.at[len(binning_table.index)-2, "Points"] = metric_special*c
+            elif metric_missing!= 'empirical':
+                binning_table.at[len(binning_table.index)-1, "Points"] = metric_missing*c
```

Comment:
Minor comment:  add space between "*", e.g., ``metric_special * c``. Keep maximum-length 79 to satisfy PEP-8. Do you need ``.index`` to compute the binning table length?